### PR TITLE
Add unwanted selector

### PR DESCRIPTION
**Pull request by Johann-S**
Hi,

This is a PR to add unwanted selector when you execute ``addAnchors``

Exemple : 
``addAnchors('h2, h3, h4','h1, .subMenu'); ``

### CODE REVIEW

**Comment by XhmikosR** [comment]
@Johann-S: can you rebase? @bryanbraun: thoughts?

File: anchor.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Copyright (c) 2015 Bryan Braun; Licensed MIT
  */
 
-function addAnchors(selector) {
+function addAnchors(selector, unWandedAnchors) {
```
Comment (cvrebert) [inline anchor.js]: Typo: Should be "Wanted" instead of "Wanded"

**Comment by bryanbraun** [comment]
@Johann-S, thanks for the request, and apologies for my late response. I thought that I had already commented on this issue :disappointed:.

@XhmikosR, I am working on a version 1.0 which will improve the architecture and make it easier to include features like this one. I have a [work-in-progress branch up](https://github.com/bryanbraun/anchorjs/tree/v1.0-wip) and I'm adding this functionality (though the API is a bit different). It will work like this:

```javascript
anchors.add('h1, h2, h3, h4,');
anchors.remove('.submenu');
```
or alternatively,
```javascript
anchors.add('h1, h2, h3, h4,').remove('.submenu');
```

 The unminified file on the WIP branch is functionally complete... you could try it out now if you wanted. I just wanted to write some automated tests and improve the documentation before it went out into the wild. If all goes well, then I expect to push it live (to npm, bower, etc) by the end of the week.

Because there are minor API changes, I can help write a PR to bootstrap to bring in the new version and resolve https://github.com/twbs/bootstrap/pull/16381.

**Comment by XhmikosR** [comment]
@bryanbraun: OK, sounds like a plan. Even better, if you could make a PR and CC me.

**Comment by Johann-S** [comment]
So what should I do ? If you want I can rebase and fix my little mistake.

**Comment by bryanbraun** [comment]
@Johann-S, I appreciate your willingness, but there's no need to rebase and correct the mistake right now. I've submitted a separate work-in-progress PR at https://github.com/bryanbraun/anchorjs/pull/34, which contains the fixes for [this issue](https://github.com/bryanbraun/anchorjs/issues/32), and other 1.0-type fixes. If you'd like to review and confirm that it meets your needs you can, but otherwise, I'm considering this specific PR closed.

**Pull request closed without merging**